pages/linux/wol.md

```diff
@@ -0,0 +1,28 @@
+# wol
+
+> Client for sending Wake-on-LAN magic packets.
+> More information: <https://sourceforge.net/projects/wake-on-lan/>.
+
+- Send a WoL packet to a device:
+
+`wol {{mac_address}}`
+
+- Send a WoL packet to a device in another subnet based on its IP:
+
+`wol --ipaddr={{ip_address}} {{mac_address}}`
```

Comment:
This confuses me a bit.  Why would you need to specify the IP address and the MAC address?  In the first example it's quite obvious that you use the MAC address to specify what host to target, but in this one it seams that you use the IP to find the host instead.  I don't understand, then, the presence of both.  Same with the below example.

Reply:
The mac is always required. I think `wol` send the network package to an specific IP address as recipient instead of a broadcast to all the network.

Reply:
Hmm. What is the MAC for?

Reply:
From [Wikipedia](https://en.wikipedia.org/wiki/Wake-on-LAN)
```
The magic packet is a frame that is most often sent as a broadcast and that contains anywhere within its payload 6 bytes of all 255 (FF FF FF FF FF FF in hexadecimal), followed by sixteen repetitions of the target computer's 48-bit MAC address, for a total of 102 bytes.
```
Sound like when you try to wake up someone repeating their name in a `while (true)`

Reply:
Nice. So: the MAC address is required and is always that of the target computer.  Thank you ✅